lib/integer.gd

```diff
@@ -990,6 +990,7 @@ DeclareGlobalFunction( "SmallestRootInt" );
 ##  <Description>
 ##  prints the prime factorization of the integer <A>n</A> in human-readable
 ##  form.
+##  See also <Ref Func="StringPP"/>.
```

Comment:
BTW, normally, I'd rewrite PrintFactorsInt to call StringPP, but there are some subtle differences between their output *sigh*. Compare also `PrimePowersInt(n)` vs `Flat(Collected(FactorsInt(n)))`...